app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,5 @@
     <string name="add_card">Add Card</string>
     <string name="add">Add</string>
     <string name="cancel">Cancel</string>
+    <string name="delete_card">Delete</string>
```

Comment:
For consistency this should probably just be named `"delete"`. We have `add` and an `add_card` strings that correspond to `Add` and `Add Card`, respectively.